### ISSUE TITLE
Only technically rootless... it does require the ability to root.

### ISSUE THREAD

**Issue by j-romchain** [urbanadventurer/Android-PIN-Bruteforce#122]
The title claims it doesn't require root, but it actually does.

**Comment by z6il** [comment]
can you add more information? , like what step that asks for root?

**Comment by j-romchain** [comment]
Sorry, forgot you mentioned nethunter in the Readme. Corrected the title of the issue. The reason I bring it up is because it is misleading for alot of the people who pay attention to such things... alot of people looking for powerfull stuff without root is because they cannot install root... and only a very small portion of people who cannot install root (locked bootloader) already have a compatible kernel for this... 

The initial reason I created this issue is that it really got my hopes up, but with a stupidly impossible to unlock bootloader on a us Samsung phone, I can't get root, and I don't have a compatible kernel, but I didn't know I had to have a compatible kernel until about a quarter down the readme.
